tools/failures.py

```diff
@@ -109,13 +110,22 @@ def format_in_table(active_jobs, master):
 
             output += '\t'
             if test in data or '' in data:
-                output += tbplnames[test]['code']
+                try:
+                    output += tbplnames[test]['code']
+                except KeyError:
+                    output += **
+                    missing_jobs.append(test)
+
                 sum_removed += 1
             else:
                 output += "--"
                 sum_remaining += 1
 
         print "%s%s" % (key, output)
+
+    if missing_jobs:
+        print "** missing jobs: %s" % ','.join(missing_jobs)
+
```

Comment:
This is a great idea, but instead of missing maybe we should call it unknown or unrecognized?